webpack_config/common.js

```diff
@@ -40,16 +40,46 @@ module.exports = {
 
   optimization: {
     splitChunks: {
-      chunks: 'all',
       cacheGroups: {
+        chunks: 'all',
+        vendor: {
+          enforce: true,
+          name: 'vendor.bundle',
+          test(mod) {
+            const excluded = `${config.chunks.individual.join('|')}|${config.chunks.electronOnly.join('|')}|${config.chunks.devOnly.join('|').replace(/\//, '[\\\\/]')}`;
+            const excludeNodeModules = new RegExp(`[\\\\/]node_modules[\\\\/]((${excluded})\.*)`);
+            const includeCommon = new RegExp(/[\\/]common[\\/]/);
+            const includeNodeModules = new RegExp(/node_modules/);
+            return mod.context
+              && includeNodeModules.test(mod.context) && !excludeNodeModules.test(mod.context) && !includeCommon.test(mod.context);
+          },
+          priority: 20
+        },
+        common: {
+          enforce: true,
+          name: 'common.bundle',
+          test: /[\\/]common[\\/]/,
+          minChunks: 2,
+          reuseExistingChunk: true,
+          priority: 10
+        },
+        vendorIndividual: {
```

Comment:
Can we find a better name?
'Individual' is not self evident. What is it suppose to represent?

Reply:
Well, this was already implemented, and I didn't want to break existing code. It's for some bigger packages to be seperated from node_modules